LeetCodeTests/DominoAndTrominoTilingTests.cs

```diff
@@ -0,0 +1,45 @@
+namespace Tests;
+using LeetCodeSolutions;
+public class DominoAndTrominoTilingTests
+{
+  private DominoAndTrominoTilingSolution _solution;
+  public DominoAndTrominoTilingTests()
+  {
+    _solution = new();
+  }
+  [Fact]
+  public void LCExample2()
+  {
+    int n = 1;
+    int desired = 1;
+    int actual = _solution.NumTilings(n);
```

Comment:
write a void test method so the call to NumTilings is only written once: it's more important for clean tests than clean implementation